src/airport_gui.cpp

```diff
@@ -107,9 +107,7 @@ struct BuildAirToolbarWindow : Window {
 		if (!gui_scope) return;
 
 		bool can_build = CanBuildVehicleInfrastructure(VEH_AIRCRAFT);
-		this->SetWidgetsDisabledState(!can_build,
-			WID_AT_AIRPORT,
-			WIDGET_LIST_END);
+		this->SetWidgetDisabledState(WID_AT_AIRPORT, !can_build);
```

Comment:
Swapped arguments?

Reply:
Oh, forgot to mention/split that... why use the function for multiple widgets when you call it for just one? So I let it use the non-plural version.

Reply:
As mentioned in chat:

```
void SetWidgetDisabledState(byte widget_index, bool disab_stat)
void SetWidgetsDisabledState(bool disab_stat, Args... widgets)
```

Someone should be hang. But not something to for in this PR :)